.github/workflows/test-updatecli.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This will help us to test these changes once it's merged and before the release happens